spring-core/src/main/java/org/springframework/util/StreamUtils.java

```diff
@@ -141,29 +139,6 @@ public static void copy(String in, Charset charset, OutputStream out) throws IOE
 		writer.flush();
 	}
 
-	/**
-	 * Copy the contents of the given InputStream to the given OutputStream.
-	 * <p>Leaves both streams open when done.
-	 * @param in the InputStream to copy from
-	 * @param out the OutputStream to copy to
-	 * @return the number of bytes copied
-	 * @throws IOException in case of I/O errors
-	 */
-	public static int copy(InputStream in, OutputStream out) throws IOException {
```

Comment:
Don't delete it. We have many legacy projects that still use this functionality. I propose to do this deprecate.

Reply:
@AntonLGVS  Restored the method I deleted, thanks.
If it's still needed to support legacy projects, how about making this method use `transferTo()` in `InputStream`?
like below
```java
public static int copy(InputStream in, OutputStream out) throws IOException {
    Assert.notNull(in, "No InputStream specified");
    Assert.notNull(out, "No OutputStream specified");

    int byteCount = (int) in.transferTo(out);
    out.flush();
    return byteCount;
}
```

Reply:
I'm not sure Spring should keep StreamUtils#copy method. For Utils such as copy method, It looks like more proper to use apache commons or guava libraries.

Reply:
@yaboong Thanks. It's a good idea to use `transferTo()` within the `copy` method.

Reply:
@bananayong Why use an external library when Spring provides a basic set of methods. If you want to delete, you must first make a deprecate to give time for the transition. Otherwise, updating the spring package will break everything.